ideascube/blog/migrations/0010_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Content = app.get_model('blog', 'Content')
+    db_alias = schema_editor.connection.alias
+    for content in Content.objects.using(db_alias).all():
+        content.summary = "<p>{}</p>".format(escape(content.summary))
```

Comment:
Hum, you're right. Then we should the `content_card.html` template. Not the migration.

All new summaries (for new contents) will have `p` root elements (because it is how tinyMCE is generating the html).